.travis.yml

```diff
@@ -22,23 +22,22 @@ jobs:
     - language: go
       # Quote the version number to avoid parsing issues like
       # https://github.com/travis-ci/gimme/issues/132.
-      go: "1.11.5"
+      go: "1.12.7"
       go_import_path: github.com/linkerd/linkerd2
+      env:
+        - GO111MODULE=on
```

Comment:
Is this because Travis forces us to have the code in `github.com/linkerd/linkerd2`?

Reply:
Yeah that's my understanding:
https://docs.travis-ci.com/user/languages/go/#go-import-path

Eventually we should be able to move away from always setting `GO111MODULE=on`, once we're confident all environments are checked out outside of the GOPATH.